### DISCUSSION TITLE
Lazy loaded list of form_builder_text_fields

### DISCUSSION THREAD

**Discussion by TimBaumgart** [danvick/flutter_form_builder#1408] [Q&A]
What's the best practice, if you have multiple text fields in e.g. a sliver list, where the first text fields are already disposed when tapping the submit button on the bottom of the page?

Not using lazy loaded lists at all is the only solution I came up with so far.

**Comment by deandreamatias** [comment]
Now I don't have a answer about that.
You found some solution @TimBaumgart ?

**Reply by TimBaumgart** [reply]
No solution other than not using lazy lists at all when working with form_builder
